src/test/groovy/com/ullink/NUnitTest.groovy

```diff
@@ -1,252 +1,253 @@
 package com.ullink
 
-import static groovy.test.GroovyAssert.shouldFail
-
 import org.gradle.api.GradleException
 import org.gradle.testfixtures.ProjectBuilder
-import org.junit.Test
-
-public class NUnitTest {
+import spock.lang.Specification
 
-    @Test
-    public void getTestInputAsList_works() {
+class NUnitTest extends Specification {
+    def "test input correctly parses lists and comma delimited strings"() {
+        expect:
         def nunit = getNUnitTask()
```

Comment:
should put in given section

Reply:
Thanks for the review! Updated in next commit.